wyvern/feature_store/feature_server.py

```diff
@@ -456,6 +460,140 @@ async def get_historical_features(
             results=final_df.to_dict(orient="records"),
         )
 
+    @app.post(f"{settings.WYVERN_HISTORICAL_FEATURES_PATH}_v2")
+    async def get_historical_features_v2(
+        data: GetHistoricalFeaturesRequestV2,
+    ) -> GetHistoricalFeaturesResponseV2:
+        # TODO: validate the data input: lengths of requests, timestamps and all the entities should be the same
+        # if "request" not in data.entities:
+        #     raise HTTPException(
+        #         status_code=400,
+        #         detail="request is required in entities",
+        #     )
+        # length_of_requests = len(data.entities["request"])
+        # length_of_timestamps = len(data.timestamps)
+        # if length_of_requests != length_of_timestamps:
+        #     raise HTTPException(
+        #         status_code=400,
+        #         detail=(
+        #             f"Length of requests({length_of_requests}) and "
+        #             f"timestamps({length_of_timestamps}) should be the same"
+        #         ),
+        #     )
+        # if length_of_requests > MAX_HISTORICAL_REQUEST_SIZE:
+        #     raise HTTPException(
+        #         status_code=400,
+        #         detail=(
+        #             f"The max size of requests is {MAX_HISTORICAL_REQUEST_SIZE}. Got {length_of_requests} requests."
+        #         ),
+        #     )
+        # for key, value in data.entities.items():
+        #     if len(value) != length_of_requests:
+        #         raise HTTPException(
+        #             status_code=400,
+        #             detail=f"Length of requests({length_of_requests}) and {key}({len(value)}) should be the same",
+        #         )
+
+        # Generate a 10-digit hex for the request
+        hex_id = secrets.token_hex(5)
```

Comment:
What's a hex?

Reply:
it's just a random hexadecimal number, which is part of the temporary/transient table name we're generating